src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -1,26 +1,41 @@
 import type { BtcTransactionUi } from '$btc/types/btc';
 import type { BtcAddress } from '$lib/types/address';
 import type { BitcoinOutput, BitcoinTransaction } from '$lib/types/blockchain';
-import { nonNullish } from '@dfinity/utils';
+import { isNullish, nonNullish } from '@dfinity/utils';
 
 export const mapBtcTransaction = ({
 	transaction: { inputs, block_index, out, hash, time },
-	btcAddress
+	btcAddress,
+	latestBitcoinBlockHeight
 }: {
 	transaction: BitcoinTransaction;
 	btcAddress: BtcAddress;
+	latestBitcoinBlockHeight: number;
 }): BtcTransactionUi => {
 	const isTypeSend = inputs.some(({ prev_out }) => prev_out.addr === btcAddress);
-	const blockIndexAvailable = nonNullish(block_index);
 	const output: BitcoinOutput | undefined = out.find(({ addr }) =>
 		isTypeSend ? addr !== btcAddress : addr === btcAddress
 	);
 
+	const confirmations = nonNullish(block_index)
+		? latestBitcoinBlockHeight - block_index
+		: undefined;
+
+	// 0 confirmations (or undefined) - status "pending"
+	// 1 - 5 confirmations - status "unconfirmed"
+	// 6 and more confirmations - status "confirmed"
+	const status =
```

Comment:
maybe these can be hardcoded as constants somewhere? it will be easier to manage, and can be re-used in the tests

Reply:
Sounds good, updated.